cookbook/core/flyte_basics/decorating_workflows.py

```diff
@@ -11,8 +11,7 @@
 Setup-Teardown Pattern
 ^^^^^^^^^^^^^^^^^^^^^^
 
-The main use case of decorating
-``@workflow``-decorated functions is when you want to establish a setup-teardown pattern that executes some task
+The main use case of ``@workflow`` decorated functions is to establish a setup-teardown pattern to execute task
```

Comment:
```suggestion
The main use case of decorating ``@workflow``-decorated functions is to establish a setup-teardown pattern to execute task
```
It's not just workflow-decorated; this page talks about decorating a workflow-decorated function.